NOTICE.txt

```diff
@@ -116,6 +116,34 @@ IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 --------------------------------------------------------------------
+Dependency: github.com/boltdb/bolt
```

Comment:
We had recently some discussions about alternatives for the FB registry file. As this is also kind of a registry, I wonder how this fits in. @urso